pagopa/io-app
# fix(IDPay): [IODPAY-173] Fix IDPay timeline operation amount sign

## Short description
This PR fixes amount signs in IDPay timeline operation details.

| Before                                                                                                                         | After                                                                                                                          |
|--------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
| <img src="https://user-images.githubusercontent.com/6160324/224295175-5837befb-240c-4bfe-b86c-8408197bace6.png" width="200" /> | <img src="https://user-images.githubusercontent.com/6160324/224295156-7202070e-ec6b-40d6-af5e-69185a957baf.png" width="200" /> |

## List of changes proposed in this pull request
- `ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx`

## How to test
- Open the details page of an IDPay initiative
- Scroll the operation list to a Transaction operation item
- Check if the values has correct sign: `-` if payment, `+` if reversal
